Source/DafnyCore/Dafny.atg

```diff
@@ -538,13 +539,16 @@ ClassDecl<DeclModifierData dmodClass, ModuleDefinition/*!*/ module, out TopLevel
   SYNC
   "{"                                            (. bodyStart = t;  .)
     {
-      (. dmod = new DeclModifierData(); .)
+      ((. dmod = new DeclModifierData(); .)
```

Comment:
I suggest formatting these lines so that the new parentheses (L542 and L545) are more conspicuous. That's probably best done by further indenting the contents between those parentheses.